website/docs/r/imagebuilder_image_pipeline.html.markdown

```diff
@@ -10,6 +10,8 @@ description: |-
 
 Manages an Image Builder Image Pipeline.
 
+~> **NOTE:** Starting with version `5.74.0`, lifecycle meta-argument [`replace_triggered_by`](https://developer.hashicorp.com/terraform/language/meta-arguments/lifecycle#replace_triggered_by) must be used in order to prevent a dependency error on destroy. 
```

Comment:
Remove trailing space to appease the markdown linter.

```suggestion
~> **NOTE:** Starting with version `5.74.0`, lifecycle meta-argument [`replace_triggered_by`](https://developer.hashicorp.com/terraform/language/meta-arguments/lifecycle#replace_triggered_by) must be used in order to prevent a dependency error on destroy.
```